server/src/main/java/io/deephaven/server/table/TableServiceGrpcImpl.java

```diff
@@ -281,6 +286,13 @@ public void whereIn(WhereInRequest request, StreamObserver<ExportedTableCreation
         oneShotOperationWrapper(BatchTableRequest.Operation.OpCase.WHERE_IN, request, responseObserver);
     }
 
+    @Override
```

Comment:
pretty sure we should remove this rpc call entirely, as dataframes are fetched as widgets, and so will go through the TicketResolverBase.AuthTransformation mechanism when fetched as an object.

Reply:
filing: https://github.com/deephaven/deephaven-core/issues/3153